src/main/webapp/script.js

```diff
@@ -85,3 +95,19 @@ function getClassID() {
   var classId = Math.floor(Math.random() * 101);
```

Comment:
Do we need this function anymore?

Reply:
Not anymore you are right, deleting now!